### DISCUSSION TITLE
for .net 3.5

### DISCUSSION THREAD

**Discussion by PepiBobanov** [Caliburn-Micro/Caliburn.Micro#835] [Q&A]
Greetings,
I have to make WPF app, that will run on .NET 3.5.
So far i found from one repo old version (1.3.2) of Cm that can run on 35, and successfully build it .
But i can't find any docs , examples, help, how to use it.
There are tutorials for new version only.
Could someone give me some tutorials , examples, chm ...
Best regards

**Comment by vb2ae** [comment]
Not sure of any documentation from versions that old.   I would look on c-sharpcorner or the codeproject for some caliburn.micro articles

https://www.c-sharpcorner.com/topics/caliburn-micro

https://www.codeproject.com/search.aspx?q=caliburn.micro+wpf&doctypeid=1%3b2%3b3%3b13%3b11%3b17
